plugins/api/discovery/src/main/java/org/apache/cloudstack/discovery/ApiDiscoveryServiceImpl.java

```diff
@@ -293,6 +310,22 @@ public ListResponse<? extends BaseResponse> listApis(User user, String name) {
         return response;
     }
 
+    private void removeQuotaApisIfQuotaEnabledButQuotaDisabledForAccount(User user, List<String> apisAllowed) {
+        if (PluginAccessConfigs.QuotaPluginEnabled.value() && !PluginAccessConfigs.QuotaAccountEnabled.valueIn(user.getAccountId())) {
+            apisAllowed.removeAll(quotaCmdList);
+            apisAllowed.add("quotaIsEnabled");
+        }
+    }
+
+    private boolean accountNotAdminAndQuotaEnabledAndQuotaAccountNotEnabledAndApiNameInQuotaCmdList(User user, String name, Account account) {
```

Comment:
```suggestion
    private boolean isAccountNotAdminAndQuotaEnabledAndQuotaAccountNotEnabledAndApiNameInQuotaCmdList(User user, String name, Account account) {
```

Reply:
though the name surely must describe the function of the method, it might be an idea to give it the name of its intent. Surely that will be shorter and thus mor readable. (not a judgement of the code, just a suggestion)